crm_phonecall_summary_restrict/views/crm_phonecall_view.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2016 Antiun Ingeniería S.L. - Jairo Llopis
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+
+<openerp>
+<data>
+
+<record id="crm_case_inbound_phone_tree_view" model="ir.ui.view">
+    <field name="name">Replace name by summary_id</field>
+    <field name="model">crm.phonecall</field>
+    <field name="inherit_id" ref="crm.crm_case_inbound_phone_tree_view"/>
+    <field name="arch" type="xml">
+        <xpath expr="//field[@name='name']" position="attributes">
+            <attribute name="name">summary_id</attribute>
```

Comment:
Don't replace, but hide the other field. This is a very bad practice, because other modules can inherit the view referencing this field.